packages/siri-sx-generator/util/siri.ts

```diff
@@ -197,6 +197,9 @@ export const getPtSituationElementFromSiteDisruption = (
                                                         OperatorName: service.operatorShortName,
                                                     },
                                                     LineRef: service.lineName.replace(/\s+/g, "_"),
+                                                    ...(!["preprod", "prod"].includes(stage)
```

Comment:
Similar to test last PR, can we create a new site constant called something like `PUBLISHED_LINE_NAME_FEATURE_FLAG` and use that here so we can find it easier when we want to release this to prod.

Reply:
done